deltachat-ios/Helper/Utils.swift

```diff
@@ -82,16 +82,25 @@ struct Utils {
     }
 
     public static func share(url: String, parentViewController: UIViewController, sourceItem: UIBarButtonItem) {
-        if let url = URL(string: url) {
-            let activityVC = UIActivityViewController(activityItems: [url], applicationActivities: nil)
-            activityVC.popoverPresentationController?.barButtonItem = sourceItem
-            parentViewController.present(activityVC, animated: true, completion: nil)
-        }
+        guard let url = URL(string: url) else { return }
+
+        Utils.share(url: url, parentViewController: parentViewController, sourceItem: sourceItem)
     }
 
     public static func share(text: String, parentViewController: UIViewController, sourceItem: UIBarButtonItem) {
-        let activityVC = UIActivityViewController(activityItems: [text], applicationActivities: nil)
-        activityVC.popoverPresentationController?.barButtonItem = sourceItem // iPad crashes without a source
+        guard let data = text.data(using: .utf8) else { return }
+
+        let tempLogfileURL = FileManager.default.localDocumentsDir.appendingPathComponent("log.txt")
```

Comment:
why not use `temporaryDirectory`, that seems to be a better fit that saving the file "forever" in the app library directory beside the database. 

```suggestion
        let tempLogfileURL = FileManager.default. temporaryDirectory.appendingPathComponent("deltachat-log.txt")
```

i also made the file name a bit more verbose (this is also the prefix we use on android)

Reply:
Good points, thank you! 

Using the temporary directory for that just didn't come to my mind, but you're right: It's a better fit.